11. errors.rst

```diff
@@ -0,0 +1,99 @@
+.. _errors:
+    
+=======
+ERRORS
+======
+
+errors are a nightmare to every programmer.
+Most of the time, the problem is a typo, followed closely by
+a misunderstanding of indentation and standards.
+|
```

Comment:
I don't think we need these `|`s instead of blank lines? There are a couple of places where those are used in this doc.